packages/components-react/src/button/Button.tsx

```diff
@@ -0,0 +1,60 @@
+import {
+  Button as UtrechtButton,
+  type ButtonProps as UtrechtButtonProps,
+} from '@utrecht/component-library-react/dist/css-module';
+import './Button.css';
+import React, { ReactElement } from 'react';
+
+type Variant = 'primary' | 'secondary' | 'tertiary';
```

Comment:
Variant mag gewoon `appearance` blijven. En de Utrecht waardes daarvoor ook. Aangezien we nog niet weten wat de uiteindelijke waarden worden (`primary` of `primary-action` of iets heel anders).